aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -623,6 +617,24 @@ void TensorIterator::mark_outputs() {
   }
 }
 
+void TensorIterator::check_overlaps() {
+  if (check_overlap_) {
+    for (int i = 0; i < num_outputs_; i++) {
+      const auto &output = operands_[i].tensor;
```

Comment:
nit: The style in this file has been leaning towards putting the & next to the auto (`const auto& output`)

Reply:
addressed